performance-storage-service/pss_project/api/models/rest/OLTPBenchRest.py

```diff
@@ -40,3 +44,11 @@ def convert_metrics_to_dict( metrics ):
         'latency': metrics.latency.__dict__
     }
     return db_formatted_metrics
+
+def convert_environment_to_dict( environments ):
+    db_formatted_environments = {
+        'os_version': environments.os_version,
+        'cpu_number': environments.cpu_number,
+        'numa_info': environments.numa_info
```

Comment:
Just curious what sort of info we are going to store here. If possible it would be nice to make the attribute name more descriptive but maybe we don't know what is going to be stored yet.

Reply:
Do you think it's a good choice if we set it similar to incremental_metrics? So we do not set the keys here and we can change the keys in the future without modifying the code.

Reply:
> Just curious what sort of info we are going to store here. If possible it would be nice to make the attribute name more descriptive but maybe we don't know what is going to be stored yet.

Do you think it's a good choice if we set it similar to incremental_metrics? So we do not set the keys here and we can change the keys in the future without modifying the code.

Reply:
Yah maybe, what sort of info will be in numa_info?

Reply:
// cpu_socket
Change it to JSON_Field.